included_software.md

```diff
@@ -72,3 +72,10 @@ The specific patch versions included will depend on when the image was last buil
 * [Doxygen](http://www.doxygen.org) - 1.8.6
 * [WASMER](https://github.com/wasmerio/wasmer)
 * [WAPM](https://github.com/wasmerio/wapm-cli)
```

Comment:
we'll want to remove these two lines if wasmer is really going

Reply:
removed